tests/features/tokens/generate.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Generate HTTP Token for Repository Service for TUF (RSTUF)
+    As an admin,
+    Admin has deployed and bootstrapped RSTUF
+
++
+    Scenario Outline: Admin uses HTTP API to generate Token
+        Given the admin has the admin password
+        And the admin has generated the admin "access_token" after login "/api/v1/token"
+        And the admin prepare the request with Method POST
+        And the admin Authorization Bearer "access_token" in the Headers
+        When the admin send the request to "/api/v1/token/new" with <scopes> and the <expires> in the payload
```

Comment:
I am not sure why we need to say that the `admin prepares the requestion with POST method`
or that the `admin uses the access_token in the authorization bearer` as separate steps.
I see them as part of the `When` portion.

I suggest you describe the HTTP API request simpler as you did in your other pr https://github.com/vmware/repository-service-tuf/pull/133/.

My suggestion here would be:
```suggestion
        When the admin sends a "POST" request to "/api/v1/token/new" with the generated "access_token" in the header and with  <scopes> and <expires> in the payload
```

Reply:
> I am not sure why we need to say that the `admin prepares the requestion with POST method` or that the `admin uses the access_token in the authorization bearer` as separate steps. I see them as part of the `When` portion.
> 
> I suggest you describe the HTTP API request simpler as you did in your other pr #133.
> 

Your suggestion is more complex if you see it from the test perspective.
The mentioned PR (#133)  has a more straightforward request (No headers required and no payload/body required) it is an simple Authentication HTTP Request.

You suggest combining multiple given; it is visible in the sentence, and you add all `And` (Given) in one sentence.

> 'When the admin sends a "POST" request to "/api/v1/token/new" with the generated "access_token" in the header **and** with  <scopes> **and** <expires> in the payload'

Another point is that all  "Given" ("And") can be reused as a fixture, as they are expected be the same across other scenarios.